library/LuaApi.cpp

```diff
@@ -2276,8 +2276,17 @@ static int constructions_designateRemove(lua_State *L)
     return 2;
 }
 
+static int constructions_findAtTile(lua_State *L)
+{
+    auto pos = CheckCoordXYZ(L, 1, true);
+    Lua::PushDFObject(L, Constructions::findAtTile(pos));
+    return 1;
+}
+
 static const luaL_Reg dfhack_constructions_funcs[] = {
     { "designateRemove", constructions_designateRemove },
+    { "findAtTile", constructions_findAtTile },
+    WRAPM(Constructions, insert),
```

Comment:
the `WRAPM` line should go in the `dfhack_constructions_module` array on line 2265